osu.Game/Graphics/Backgrounds/BeatmapBackgroundWithStoryboard.cs

```diff
@@ -19,16 +22,24 @@ public BeatmapBackgroundWithStoryboard(WorkingBeatmap beatmap, string fallbackTe
         [BackgroundDependencyLoader]
         private void load()
         {
-            LoadComponentAsync(new DrawableStoryboard(Beatmap.Storyboard)
-                {
-                    Clock = new InterpolatingFramedClock(Beatmap.Track),
-                },
-                loaded =>
-                {
-                    AddInternal(loaded);
-                    if (Beatmap.Storyboard.ReplacesBackground)
-                        Sprite.FadeOut(300, Easing.OutQuint);
-                });
+            var storyboard = new Storyboard { BeatmapInfo = Beatmap.BeatmapInfo };
```

Comment:
This is wrong. You need to use the beatmap's storyboard. Things like `UseSkinSprites` are no longer set correctly.

Reply:
Trying to add or remove elements from the working beatmap itself would also affect the storyboard when it is consumed later in gameplay. I can however pass in the `UseSkinSprites` value by:
```suggestion
            var storyboard = new Storyboard 
            {
                BeatmapInfo = Beatmap.BeatmapInfo,
                UseSkinSprites = Beatmap.Storyboard.UseSkinSprites,
            };
```

Reply:
That doesn't sound like it will scale.

Reply:
I think what you're going to want is to make a derived `DrawableStoryboard` which handles the filter operations internally. So you don't need to mutate the `Storyboard` in the first place.

Reply:
I've done a bit more research on how the elements are created.

The elements in the `DrawableStoryboard` are deeply nested inside the `LayerManagementContainer`. The hierarchy looks like this:

```
DrawableStoryboard
├─ DrawableStoryboardLayer
│  ├─ LayerElementContainer
│  │  ├─ DrawableStoryboardSprite
```

In short, there is no easy access to the drawable storyboard elements in the first place without making major changes. I'm rather unsure if this is the intended direction or not.

Reply:
So to keep things simple then:

- There's no real reason to remove the fail layer right? It's not going to display either way? If so ignore it and just let it be.
- To mute sounds, you should be able to nest the whole `DrawableStoryboard` inside an `AudioContainer` and set the volume adjustment on that to 0.